src/styles/blocks/watch.scss

```diff
@@ -0,0 +1,22 @@
+.watch {
+  height: $watch-size;
+  width: $watch-size;
+  border: 5px dotted black;
+  border-radius: 50%;
+  align-self: center;
+  display: flex;
+  justify-content: center;
+
+  &__center {
+    height: $watch-size * 0.03;
+    width: $watch-size * 0.03;
+    background-color: black;
+    border: 5px dotted black;
+    border-radius: 50%;
+  }
+
+  &__center--alignment {
```

Comment:
When you move this style into just watch__center it won't be enough ? Think you don't need this class anymore :)